<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>185</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>185</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>185</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>生成随机数</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上限值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下限值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>生成数量：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_pos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>正序</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_rev">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>逆序</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_confu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>混乱</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_max">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_min">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_num">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sure</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>btn_sure_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_sure_click()</slot>
 </slots>
</ui>
